chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,611 @@
+then you should create a \textbf{master dataset}\sidenote{
+	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}}
+for each of those units of observation and include that in the required datasets.
+The master dataset should include an ID variable
+that can be used to link or merge all datasets with that unit of observation.
+It should also list project wide time-invariant information
+like sampling and treatment status.
+Additionally, any unit of observation
+that will be used in sampling or treatment assignment
+should have a master dataset,
+and that dataset should be used when sampling or assigning treatment.
+
+\section{Taking research design into account when planing your data work}
+
+The most important steps in data work planning
+to correctly translate a research design into data work
+are sampling and treatment assignment.
+Therefore, understanding and correctly implementing sampling and treatment assignment
+is essential to ensuring that data is acquired properly and that research
+is correctly carried out in the field, so that the results
+are valid interpretations of your research question.
+
+Not all projects need to do sampling and treatment assignment.
+Sampling is almost always required when you collect your own data,
+as it is often prohibitively expensive to collect data on all individuals.
+However, if the data you use is, for example,
+the tax filings of all companies in a country,
+then you can estimate the treatment effect
+using the full population and not a sample of it.
+Such big data might need a cluster of computers to process,
+so you might still want to know how make a representative sample,
+so that you can develop your code on a local computer
+before running it on an expensive computer cluster.
+
+Treatment assignment is only applicable if your research design is experimental,
+and the research team can control which individuals receive the treatment.
+While you might not yet have a population dataset to sample from,
+or a complete list of eligible observation to randomize assignment on,
+this is still the time to understand those tasks
+and what you need to think about as you acquire those datasets.
+
+Randomization is used to ensure that a sample is representative and
+that the treatment and control groups are statistically indistinguishable
+after treatment assignment.
+Randomization in statistical software is non-trivial
+and its mechanics are unintuitive for the human brain.
 ``True'' randomization is also nearly impossible to achieve for computers,
 which are inherently deterministic.\sidenote{
-  \url{https://www.random.org/randomness}}
+	\url{https://www.random.org/randomness}}
+The principles of randomization we will outline
+apply not just to random sampling, random assignment,
+but they apply to all statistical computing processes that have a random components,
+such as simulations and bootstrapping.
+
+The next section discuss how a random number
```

Comment:
```suggestion
The next subsection discuss how a random number
```

Reply:
technically correct yes, but is the term `sub-section` meaningful to the reader?

Reply:
Let's push this to final review, make a decision, and apply consistently across the book. i think subsection is clear but let's check with others.

Reply:
Mentioned in issue here: https://github.com/worldbank/d4di/issues/455